client/asset/btc/btc.go

```diff
@@ -2350,13 +2416,48 @@ func (btc *baseWallet) AuditContract(coinID, contract, txData dex.Bytes, rebroad
 
 	return &asset.AuditInfo{
 		Coin:       newOutput(txHash, vout, uint64(txOut.Value)),
-		Recipient:  receiver.String(),
+		Recipient:  btc.stringifyAddress(receiver),
 		Contract:   contract,
 		SecretHash: secretHash,
 		Expiration: time.Unix(int64(stamp), 0).UTC(),
 	}, nil
 }
 
+// AuditContract retrieves information about a swap contract from the provided
+// txData. The extracted information would be used to audit the counter-party's
+// contract during a swap. Since this wallet is backed by a full node, txData
+// may be empty to attempt retrieval of the transaction output from the network.
+func (btc *ExchangeWalletFullNode) AuditContract(coinID, contract, txData dex.Bytes, rebroadcast bool) (*asset.AuditInfo, error) {
+	if len(txData) != 0 {
+		return btc.baseWallet.AuditContract(coinID, contract, txData, rebroadcast)
```

Comment:
I think this `ExchangeWalletFullNode` method was left over from rejected code from a [previous PR of mine](https://github.com/decred/dcrdex/pull/1553#discussion_r843386210) (on iteration https://github.com/decred/dcrdex/commit/bae9942753de9d56596434db0c34ce89da7faeec).  I guess this zcash PR was based on top of it at one point, and then you rebased the whole branch on master instead of using `--onto` and just your commits.

Or did this method actually turn out to be helpful?